css-two-column-layout/styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Karla, sans-serif;
+}
+
+.header {
+  text-align: center;
+  margin-bottom: 45px;
+}
+
+p.headingText {
+  width: 400px;
+  margin: 0 auto;
+}
+
+p {
+  font-size: 9px;
+  line-height: 15px;
+}
+
+.container {
+  max-width: 600px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-fourth {
+  width: calc(200% / 4);
```

Comment:
This doesn't add up to a quarter, it adds up to a half 🤔 . You should rename this to `column-half`